wiki/en/Installation-for-Windows.md

```diff
@@ -22,9 +22,13 @@ Upgrading? You may want to [back up your configuration](Software-Manual#backing-
 
 _Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._
 
+If you don't want to use ASIO, you can also use the (JACK version described below)(#JACK).
+
 ***
 
-## Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) 
+## ASIO
```

Comment:
@gilgongo since we link ASIO in the app, we need to have an anchor with the id ASIO on this page.

Reply:
Sorry what do you mean? We have the ASIO rights statement in the site footer now if that's what you mean?

Reply:
No. The Windows installer links to the headline ASIO via a HTML ID (anchor). So that means we must not name the headline anything other than ASIO.

Reply:
Oh OK sure.